loop/storage/redis.js

```diff
@@ -870,24 +800,15 @@ RedisStorage.prototype = {
     if (isUndefined(hawkIdHmac, "hawkIdHmac", callback)) return;
 
     var self = this;
-    self._client.expire(
-      'userid.' + hawkIdHmac,
-      self._settings.hawkSessionDuration,
-      function(err) {
-        if (err) return callback(err);
-        self._client.expire(
-          'hawk.' + hawkIdHmac,
-          self._settings.hawkSessionDuration,
-          function(err) {
-            if (err) return callback(err);
-            self._client.expire(
-              'hawkuser.' + hawkIdHmac,
-              self._settings.hawkSessionDuration,
-              callback
-            );
-          }
-        );
-      });
+    var multi = self._client.multi();
+
+    multi.expire('userid.' + hawkIdHmac,
```

Comment:
nit: for readability, I would put this `self._settings.hawkSessionDuration` into a temporary variable (`hawkSessionDuration` ?)